<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="napravi">
          <property name="text">
           <string>napravi</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QCheckBox" name="checkBox">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>CheckBox</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_2">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>CheckBox</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_3">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>CheckBox</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_4">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>CheckBox</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_5">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>CheckBox</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_6">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>CheckBox</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_7">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>CheckBox</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_8">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>CheckBox</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_9">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>CheckBox</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_10">
                <property name="text">
                 <string>CheckBox</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCheckBox" name="checkBox_11">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>nije potrebno</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_12">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>nije potrebno</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_13">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>nije potrebno</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_14">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>nije potrebno</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_15">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>nije potrebno</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_16">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>nije potrebno</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_17">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>nije potrebno</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_18">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>nije potrebno</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_20">
              <property name="text">
               <string>nije potrebno</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_19">
              <property name="text">
               <string>nije potrebno</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="prijavi">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>prijavi</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Uradjeno : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Neuradjeno : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Nije potrebno :</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
